### PR TITLE
[12.0][MIG] account_voucher_killer

### PR DESCRIPTION
**Pull request by astirpe**
Moved from https://github.com/OCA/bank-payment/pull/605

### CODE REVIEW

File: account_voucher_killer/view/invoice_view.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="invoice_payment_group_customer_invoice" model="ir.ui.view">
+      <field name="name">Hide payment in invoice</field>
+      <field name="model">account.invoice</field>
+      <field name="inherit_id" ref="account.invoice_form" />
+      <field name="arch" type="xml">
+        <xpath expr="//button[@name='%(account.action_account_invoice_payment)d']"
```
Comment (ThomasBinsfeld) [inline account_voucher_killer/view/invoice_view.xml:9]: Nitpicking: 
```suggestion
        <button name="%(account.action_account_invoice_payment)d"
```
(same for two other xpath's)

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by Saran440** [comment]
Function test :+1:

**Comment by astirpe** [comment]
Thanks @Saran440 could you please go to "Files changed" -> "Review changes" and then "Approve" ?

**Review by Saran440 — APPROVED** [review]

**Pull request closed without merging**